Source/WebKitLegacy/mac/WebView/WebFeature.h

```diff
@@ -59,6 +59,7 @@ typedef NS_ENUM(NSUInteger, WebFeatureCategory) {
     WebFeatureCategoryAnimation,
     WebFeatureCategoryCSS,
     WebFeatureCategoryDOM,
+    WebFeatureCategoryHTML,
```

Comment:
I need to forward-declare this value, and likely others in the future if we add them. To make that possible, can we add the new value to the end and provide an explicit value for each of these?